seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -19,4 +19,7 @@ ethereum,Curve,Regular,2022-09-03,0x0fc5910c411bdbbf1827fa278c22b3351892e8931e7c
 ethereum,Curve,Regular,2022-09-04,0x4ca41e0f0700f357111aaae66a85b0def53b00a6ca02145e6e6ec00775621df5,537,0x6c3f90f043a72fa612cbac8115ee7e52bde6e490,79307009658375330225086,0x5f98805a4e8be255a32880fdec7f6728c6568ba0,7.78328E+22
 ethereum,Curve,Factory V1 Meta,2022-09-23,0x182e11e393388ea9365b1e566bfc66754bbdfb7f83344f6e1e4f05cde97ea66b,11,0x6c3f90f043a72fa612cbac8115ee7e52bde6e490,429398385988372154944,0xa693b19d2931d498c5b318df961919bb4aee87a5,10259457322
 ethereum,Curve,Factory V1 Plain,2022-09-19,0xe31b85292938984c61b4a9d8d2eb781f20bfa22c068a380a67fce94ef8ec450c,60,0xd533a949740bb3306d119cc777fa900ba034cd52,742474512665704726710,0x62b9c7356a2dc64a1969e19c23e4f579f9810aa7,774567425737246266396
-ethereum,Curve,Factory V2,2022-09-03,0x02fe546401dd52e07528532ed1ac4ed1d4f525118836f71a4014282fb6a06839,4,0xaf5191b0de278c7286d6c7cc6ab6bb8a73ba2cd6,51356173218735436675801,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,33333000000
\ No newline at end of file
+ethereum,Curve,Factory V2,2022-09-03,0x02fe546401dd52e07528532ed1ac4ed1d4f525118836f71a4014282fb6a06839,4,0xaf5191b0de278c7286d6c7cc6ab6bb8a73ba2cd6,51356173218735436675801,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,33333000000
+arbitrum,uniswap,3,2022-10-11,0x88a863ee12e4258c93bf41651853093f071d53facf0cbe0ae7b2bad040928f50,3,0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8,3198364498,0x82aF49447D8a07e3bd95BD0d56f35241523fBab1,2500000000000000000
```

Comment:
looks like you may have flipped token bought/sold here? or is it flipped in the code/model?
```
with
  unit_test as (
    select
      lower(m.token_bought_address) as model_token_bought,
      lower(seed.token_bought_address) as seed_token_bought,
      lower(m.token_sold_address) as model_token_sold,
      lower(seed.token_sold_address) as seed_token_sold,
      case
        when lower(m.token_bought_address) = lower(seed.token_bought_address) then True
        else False
      end as token_bought_test
    from
      dbt_jeff_uniswap_v3_arbitrum.trades m
      inner join dbt_jeff_test_data.dex_trades_seed seed on m.tx_hash = seed.tx_hash
      and m.evt_index = seed.evt_index
      and m.block_date = seed.block_date
      and m.blockchain = seed.blockchain
      and m.project = seed.project
      and m.version = seed.version
  )
select
  *
from
  unit_test
where
  token_bought_test = False
```

this is why we try to add the tests 👍

Reply:
also addresses should be lower everywhere (they aren't in the seed here). to be safe, i've added lower() function in the generic test applied

Reply:
I had them flipped in the seed file.  I swapped them and made lower case